labs/accessing-optimizely-event-data/metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Accessing Optimizely Enriched Event Data
+summary: This lab contains a handy CLI for exploring and downloading Optimizely Enriched Event Data.
+revisionDate: '2020-07-20'
+labels:
+  - data
+  - analysis
+  - odata
```

Comment:
Does this stand for Optimizely Data?

Reply:
Whoops, this was another name candidate.  I replaced it with `oevents`